errors/invalid-images-config.md

```diff
@@ -33,6 +33,8 @@ module.exports = {
     dangerouslyAllowSVG: false,
     // set the Content-Security-Policy header
     contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
+    // when true, every image will be unoptimized
+    unoptimized: false,
```

Comment:
Does this need to be broken out into the experimental field as well?

Reply:
Nice catch. Also needed to move remotePatterns 👍  [63dbeda](https://github.com/vercel/next.js/pull/37698/commits/63dbeda74940704f0d6d4a283308d331cc09017b)